docs/book/src/user/quick-start.md

```diff
@@ -740,6 +740,38 @@ clusterctl init --infrastructure vsphere
 For more information about prerequisites, credentials management, or permissions for vSphere, see the [vSphere
 project][vSphere getting started guide].
 
+{{#/tab }}
+{{#tab Proxmox}}
+
+First, we need to add the Proxmox infrastructure provider and the IPAM provider to your clusterctl config file `~/.cluster-api/clusterctl.yaml`:
+```yaml
+providers:
+  - name: in-cluster
+    url: https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/releases/latest/ipam-components.yaml
+    type: IPAMProvider
+  - name: proxmox
+    url: https://github.com/ionos-cloud/cluster-api-provider-proxmox/releases/latest/infrastructure-components.yaml
```

Comment:
You might want to add the Proxmox provider to clusterctl so you don't need this step.

But as long we're waiting for the IPAM provider to be added it doesn't significantly simplify these steps anyway :)

Reply:
yeah, I'm looking to add it in clusterctl, but waiting for IPAM as well.

Reply:
Feel free to add in independently of the IPAM provider to clusterctl if you want

Reply:
so, you mean I can add the Proxmox in defaults?
https://github.com/kubernetes-sigs/cluster-api/blob/main/cmd/clusterctl/client/config/providers_client.go

and only add ipam into clusterctl config

Reply:
Exactly

Reply:
done